src/i18n/locales/en.json

```diff
@@ -498,6 +498,29 @@
         "pause": "pause",
         "viewQueue": "view queue"
     },
+    "releaseType": {
+        "primary": {
+            "album": "$t(entity.album_one)",
+            "broadcast": "broadcast",
+            "ep": "ep",
+            "other": "other",
+            "single": "single"
+        },
+        "secondary": {
+            "audiobook": "audiobook",
+            "audio drama": "audio drama",
+            "compilation": "compilation",
+            "dj-mix": "dj mix",
+            "demo": "demo",
+            "field recording": "field recording",
+            "interview": "interview",
+            "live": "live",
+            "mixtape/street": "mixtape",
+            "remix": "remix",
+            "soundtrack": "soundtrack",
+            "spokenword": "spoken word"
+        }
```

Comment:
I'm not a fan of using the raw values as keys here. It makes it inconsistent with the rest of the json schema.

Can you rename them to use camelCase and then add a secondary mapping from the i18n key to the releaseType?